gems/pending/fs/xfs/allocation_group.rb

```diff
@@ -115,8 +115,8 @@ def initialize(stream, _agno, sb)
       @stream.seek(AG_INODEINFO_SIZE, IO::SEEK_CUR)
       @agfl                   = AG_FREELIST.decode(@stream.read(AG_FREELIST_SIZE))
       @stream.seek(-(AG_FREESPACE_SIZE + AG_INODEINFO_SIZE + AG_FREELIST_SIZE))
-      @allocation_group_block = MemoryBuffer.create(sb['block_size'])
-      @allocation_group_block = @stream.read(sb['block_size'])
+      @allocation_group_block                      = MemoryBuffer.create(sb['block_size'])
+      @allocation_group_block[0, sb['block_size']] = @stream.read(sb['block_size'])
```

Comment:
@Fryguy I was under the impression that other fs implementations also used this in a like manner, but upon further review, it seems that they are only using it to either get empty buffers or buffers that may be partially empty.  This case is not the same and therefore I will remove the line.  Nothing like a 2-line PR that is 50% wrong.... :-(